lib/game/player.ts

```diff
@@ -75,6 +78,60 @@ export class Player extends EventEmitter {
 		}
 	}
 
+	/**
+	 * This is only used for tests, and simulates time by running the physics
+	 * loop as well as the animations every frame.
+	 * @param dTime Time to simulate in ms
+	 */
+	public simulateTime(dTime: number) {
```

Comment:
minor: personal taste - I like units in the variable, less thinking when look at it

Reply:
True, but in this case it's documented right above and the function is like five lines :)